wazo_confd/plugins/user/schema.py

```diff
@@ -17,7 +17,7 @@
 
 class UserSchema(BaseSchema):
     id = fields.Integer(dump_only=True)
-    uuid = fields.String(dump_only=True)
+    uuid = fields.String()
```

Comment:
This looks quite dangerous to me: could an API user change the UUID of a user after its creation?